pkg/restapi/operation/operations_test.go

```diff
@@ -190,6 +192,20 @@ func (m *mockEDVStore) Query(*models.Query) ([]string, error) {
 	return []string{"docID1", "docID2"}, nil
 }
 
+type mockUUIDBuilder struct {
+	testUUIDString string
+}
+
+// mock uuid().New() to replicate generating duplicated uuid vault names
+func (m *mockUUIDBuilder) New() uuid.UUID {
+	id, err := uuid.Parse(m.testUUIDString)
+	if err != nil {
+		panic(err)
```

Comment:
In general we want to avoid adding direct `panic` calls to our code if we can...

Also, you have an unnecessary UUID->string->UUID conversion going o. Instead of defining `testUUIDString` as a string, just make it a `UUID` type (and of course remove the `string` from the name). Then, simply pass in `uuid.New()` instead of `uuid.New().String()` to initialize the variable. Now this method becomes a simple return: `return m.testUUID`, and the error check and panic go away.